resources/views/pages/expense/index.blade.php

```diff
@@ -32,7 +33,24 @@
       <div class="col-md-8 col-lg-12 section-heading">
         <h1 class="section-heading-title">Expense Report</h1>
         <p class="section-heading-paragraph">Expense report gives an insight to how much is being spent by the federal government on a daily basis and how much is spent in major sectors in Nigeria.</p>
-      </div>
+        <h5>Subscribe to daily Expense Report</h5>
+        <span>
+            <form method="POST" action="{{ url('expense/report')}}">
+              @csrf
+              <input type="text" class="form-control" name="email" placeholder="Enter Your Email" style="width: 50%;">
+              <br>
+              <select name="subscription_type" id="" class="form-control" style="width: 50%;">
+                  <option value="">Select The Report type</option>
+                  <option value="ministry">Ministry</option>
+                  <option value="contract">Contract</option>
+                  <option value="daily&expense">Daily Expense</option>
+                  <option value="payment">Payments W/0 description</option>
```

Comment:
Please change this to something more descriptive. `Payments without subscription` is fine or something else the user can relate with

Reply:
will do that thanks

Reply:
@goodhands  fixed all what you said